File: Beecrowd/8-Geometria_Computacional/2830-Balao++-2/2830.c
#include <stdio.h>


int main(){

    double radix, gas;
    
    scanf("%lf%lf", &radix, &gas);
    
    double area =  (4.0/3.0) * (3.1415*(radix*radix*radix));
    
    printf("%d\n",(int)(gas/area));
    
    return 0;
}

File: Beecrowd/8-Geometria_Computacional/2830-Balao++-2/2830.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        double radix = scanner.nextDouble();
        double gas = scanner.nextDouble();
        
    
        double area = (4.0 / 3.0) * (3.1415 * (radix * radix * radix));
        
        System.out.println((int)(gas / area));

    }
}
